Assignment1/Co2Emission.ts

```diff
@@ -0,0 +1,41 @@
+const CO2_EMISSIONS: { [key: string]: number } = {
+  inbox: 0.02,
+  sent: 0.05,
+  spam: 0.01,
+};
+
+const emailData: { [key: string]: number } = {
+  inbox: 500,
+  sent: 200,
+  spam: 100,
+};
+
+interface Entity {
+  email: string;
+}
+
+interface CarbonFootprint {
+  emailId: string;
+  source: string;
+  inbox: string;
```

Comment:
variable names - inbox, spam, sent are not descriptive and do not give any idea about what value they hold